Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeAMSIConfigurationState.ps1

```diff
@@ -0,0 +1,76 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\..\Helpers\Invoke-CatchActions.ps1
+Function Get-ExchangeAMSIConfigurationState {
+    [CmdletBinding()]
+    param ()
+
+    begin {
+        Function Get-AMSIStatusFlag {
+            [CmdletBinding()]
+            [OutputType([bool])]
+            param (
+                [Parameter(Mandatory = $true)]
+                [object]$AMSIParameters
+            )
+
+            try {
+                Switch ($AMSIParameters.Split("=")[1]) {
+                    ("False") { return $false }
+                    ("True") { return $true }
+                    Default { return $null }
+                }
+            } catch {
+                throw
```

Comment:
Add a `Write-Verbose` in here to help determine where we are at if we do `throw` out of this. Maybe even add `$AMSIParameters` to the wording

Reply:
We now log the following before throwing an exception:

`Write-Verbose "Ran into an issue when calling Split method. Parameters passed: $AMSIParameters"`